crates/pop-cli/src/commands/new/contract.rs

```diff
@@ -64,8 +67,8 @@ impl NewContractCommand {
 	}
 }
 
-fn is_template_supported(contract_type: &ContractType, template: &Template) -> Result<()> {
-	if !template.matches(contract_type) {
+fn is_template_supported(contract_type: &ContractType, template: &ContractTemplate) -> Result<()> {
+	if !contract_type.matches(template) {
```

Comment:
Note, before individual templates were able to use the method matches to see if it belongs to a TemplateType; "does template type match template". This PR has flipped this functionality and now matches the template to the template type; "does this template match the template type".

Reply:
No objections apart from the naming. Suggested 'provides' or 'supports' elsewhere in comments.